cli/pkg/credentialmanager/handler.go

```diff
@@ -171,12 +171,14 @@ func getCurrentContextFromKubeConfig() error {
 
 	fileContent, err := file.ReadFile(kubeconfig)
 	if err != nil {
-		return err
+		logging.PrintLog("Warning: could not open KUBECONFIG file: "+err.Error(), logging.InfoLevel)
+		return nil
```

Comment:
I guess by returning `nil` here, 
```
		err := getCurrentContextFromKubeConfig()
		if err != nil {
			log.Fatal(err)
		}
		checkForContextChange(cliConfigManager, autoApplyNewContext)
```
won't exit, and therefore Issue #2834 will be solved?

Reply:
exactly. in this case, the current kubernetes context will be set to `""`, and the CLI will proceed to use the currently active credentials